www/main/navigator/single-display/singleDisplayFooter.jsx

```diff
@@ -63,6 +68,12 @@ export const singleDisplayFooter = () => {
       >
         <i className="fa fa-ellipsis-h" />
       </button>
+      <button
+        className={`tab-switch ${singleDisplayActiveTab === 'favorite' ? 'active' : ''}`}
```

Comment:
classNames function is recommended here veerji.

Reply:
Done